components/compliance-service/api/reporting/server/server.go

```diff
@@ -63,6 +63,48 @@ func (srv *Server) ListReports(ctx context.Context, in *reporting.Query) (*repor
 	return &reports, nil
 }
 
+// ListReports returns a list of reports based on query
+func (srv *Server) ListReportIds(ctx context.Context, in *reporting.Query) (*reporting.ReportIds, error) {
```

Comment:
i know the pr is still marked as wip...but i assume you'll be adding a test or two for this new fancy endpoint?

Reply:
yep.. i got the tests in now.. will take this out of WIP